capa/features/extractors/dnfile/helpers.py

```diff
@@ -211,7 +223,13 @@ def get_dotnet_managed_methods(pe: dnfile.dnPE) -> Iterator[DnType]:
                 # remove get_/set_
                 method_name = method_name[4:]
 
-            yield DnType(token, typedef.TypeName, namespace=typedef.TypeNamespace, member=method_name, access=access)
+            typedef_name = []
+            typedefname = (typedef.TypeName,)
+            typedefnamespace = typedef.TypeNamespace
```

Comment:
Can we move these declarations to an `else` clause that is attached to the `if` statement below? This avoids creating and then immediately overwriting these objects when nested classes are encountered.